schema.sql

```diff
@@ -15,5 +15,6 @@ CREATE TABLE items (
   fit varchar(280),
   sizes varchar(280),
   colors varchar(280),
+  price varchar(15),
```

Comment:
You may want to consider setting price as an `INT` and formatting it client-side. Although it's not required for this project, price as `VARCHAR `may lead to unexpected behavior when manipulating prices (eg. sales, discounts, etc).

Reply:
Definitely feels hacky, right? This is going on my list of tech-debt tasks.